comic_pile/session.py

```
@@ -105,98 +102,89 @@ def get_or_create(db: Session, user_id: int) -> SessionModel:
+            active_session = result.scalars().first()
 
             if active_session:
                 return active_session
 
-            with _session_creation_lock:
+            async with _session_creation_lock:
                 try:
-                    db.execute(text("SELECT pg_advisory_xact_lock(12345)"))
+                    await db.execute(text("SELECT pg_advisory_xact_lock(12345)"))
                 except Exception:
                     pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)